dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/Application.cs

```diff
@@ -28,6 +29,9 @@ public class Application<TState, TTurnStateManager> : IBot
         where TState : ITurnState<StateBase, StateBase, TempState>
         where TTurnStateManager : ITurnStateManager<TState>, new()
     {
+        private static readonly string CONFIG_FETCH_NAME = "config/fetch";
```

Comment:
How about `CONFIG_FETCH` or `TEAMS_CONFIG_FETCH`?

Reply:
Updated to `CONFIG_FETCH_INVOKE_NAME` to sync with JS part :)